### PR TITLE
Add webiste revamp idea

### PR DESCRIPTION
**Pull request by ArthurSens**

### CODE REVIEW

**Review by simonpasquier — COMMENTED** [review]
From my POV, there can be different work streams around documentation:
1. Update the documentation framework to the latest versions (e.g. doks, Hugo, ...)
2. Review the documentation at https://github.com/prometheus-operator/prometheus-operator/tree/main/Documentation not available at https://prometheus-operator.dev and incorporate what's missing (after evaluation).
3. (a bit related to the previous one) review the current content and suggest a more cohesive organization/plan.

I'm not sure if we need 3 "ideas" or not...

**Comment by ArthurSens** [comment]
I don't think we need 3 projects, but yeah... the description could be re-organized to mention those three workforces

**Comment by ArthurSens** [comment]
PTAL

**Review by nicolastakashi — APPROVED** [review]

File: mentoring/gsoc/2024/project_ideas.md [hunk 1]
```diff
@@ -26,3 +26,16 @@ If you are a Prometheus-Operator team member and consider mentoring during the G
...
 ## Ideas
+
+### Webiste/Documentation revamp
+
+- Description: We maintain all of our documentation on our website https://prometheus-operator.dev/ . The website was created a couple of years ago and not much has changed ever since, from project documentation exposed by the website or even the framework used to build it. This project is about different workstreams related to our website and documentation:
+  - Update framework, such as doks and hugo, and propose automation to keep it up to date.
+  - Review the documentation available in our [main git repository](https://github.com/prometheus-operator/prometheus-operator/tree/main/Documentation) that is not available in the website, and incorporate it to the website.
+  - Review and propose a more coherent content organization.
+  - Propose changes to our development process to help us keep documentation up to date.
+- Recommended Skills: Tech writing, Prometheus, Alertmanager
```
Comment (simonpasquier) [inline mentoring/gsoc/2024/project_ideas.md:37]:
```suggestion
- Recommended Skills: Technical writing, Prometheus, Alertmanager
```

File: mentoring/gsoc/2024/project_ideas.md [hunk 2]
```diff
@@ -26,3 +26,16 @@ If you are a Prometheus-Operator team member and consider mentoring during the G
...
+- Expected project size: small
```
Comment (simonpasquier) [inline mentoring/gsoc/2024/project_ideas.md:38]: medium?

**Pull request merged**